YukonVaccinationVerifier/YukonVaccinationVerifier/ViewController.swift

```diff
@@ -320,6 +320,10 @@ class ViewController: UIViewController {
         view.addSubview(logoImageView)
         
         logoImageView.image = UIImage(named: "onCameraLogo")
+        
+        // Add Business Guidance Links
+        
+        BusinessGuidance.setupView(in: self, links: [BusinessGuidanceLink(title: "COVID-19 information", url: "https://yukon.ca/en/covid-19-information"), BusinessGuidanceLink(title: "Data collection notice", url: nil)])
```

Comment:
This function will get called more than one time.
Does that mean the 'BusinessGuidance' view will get stacked upon leading to memory issues?
Please don't hard-code strings and links. We need to localize and also definitions in a single place would help us going forward.
Also, as this is getting shown at the bottom so please make sure it respects the safe area in both the iPads and iPhones starting iOS 11.

Reply:
Also the camera needs to be inactive when we display the Safari controllers.

Reply:
this function is called the same ways other get called, like this "view.addSubview(logoImageView) line: 320"
Sure, the code displays the view respect the safe area.
hard code strings left because you were implementing the localization, so I left it like other existing strings!